src/bmm-skills/4-implementation/bmad-retrospective/scripts/sprint_status.py

```
@@ -0,0 +1,276 @@
+            "action_items_added": items_added,
+            "last_updated": last_updated,
+        }
+    )
+
+
+def _restore(path, original_bytes):
+    try:
+        with open(path, "wb") as fh:
+            fh.write(original_bytes)
+    except Exception:  # noqa: BLE001 - best-effort restore
+        pass
```

Ruff (0.15.21)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)